src/shoobx/wfmc/process.py

```diff
@@ -675,12 +686,8 @@ def start(self, *arguments):
         evaluator = interfaces.IPythonExpressionEvaluator(self)
 
         # Assign data defaults.
-        for id, datafield in definition.datafields.items():
-            val = None
-            if datafield.initialValue:
-                val = evaluator.evaluate(datafield.initialValue)
-            setattr(data, id, val)
-
+        vals = getInitialDataFieldsValues(definition.datafields, evaluator)
+        data.__dict__.update(vals)
```

Comment:
This is not necessarily equivalent to calling `setattr()`, because it will skip `__setattr__` hooks (which may mark object as "changed" to be persisted  to db, for example). Let's rather do it through `setattr`.